pre-commit.sh

```diff
@@ -34,10 +34,19 @@ else
     changed_files=`git diff --name-only "${base_commit}..HEAD"`
     dirty_files=`git ls-files -m`
     files_to_check="$((echo "$changed_files"; echo "$dirty_files") | grep '\.py$')"
-    if [ -n "$files_to_check" ]; then
-        echo "$files_to_check" | xargs yapf --in-place
-    else
+    if [ -z "$files_to_check" ]; then
         echo 'nothing to run yapf on after all'
+    else
+        echo -n 'running yapf... '
+
+        echo "$files_to_check" | while read file
```

Comment:
to fix problem with `wait`, please put the opening parenthesis before `while read files` and closing parenthesis after `wait`:
```
echo "$files_to_check" | (while read file
do
    yapf --in-place "$file" &
done
wait
)
echo 'done'
```